apps/bookmark/test/unit/mock_web_manifest_helper.js

```diff
@@ -3,7 +3,11 @@
 /* exported MockWebManifestHelper */
 var MockWebManifestHelper = {
   getManifest: function(manifestURL) {
-    return Promise.resolve({'short_name': 'App', 'name': 'My App'});
+    return Promise.resolve({
+      'short_name': 'App',
+      'name': 'My App',
+      'displayName': 'App'
```

Comment:
This is probably masking over genuine test failures caused by the change to bookmark_editor.js